examples/monolithic_build_multilevel_native/mlkem_native_all.c

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2024 The mlkem-native project authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define MLKEM_NATIVE_MULTILEVEL_BUILD
+
+/* Three instances of mlkem-native for all security levels */
+
+/* Include level-independent code */
+#define MLKEM_NATIVE_MULTILEVEL_BUILD_WITH_SHARED 1
```

Comment:
Why is this set as 1?

Reply:
So it doesn't give a warning with how `-D...` in the CFLAGS is done